### PR TITLE
Player hud

### PR DESCRIPTION
**Pull request by aleaiudi**

### CODE REVIEW

File: Assets/Lobby-NetworkClient/ClientUtilities.cs [hunk 1]
```diff
@@ -337,20 +342,28 @@ public static void ParseClientDodged(Packet packet)
...
         if (ClientManager.IsClientJoined(clientId))
-        {
-            if (packet.TimeStamp - ClientManager.GetClient(clientId).LastTimeStamp > 5.0f)
-            {
-                Debug.Log("PACKET OUT OF TIME STAMP");
-                return;
-            }
+        {       
+            Debug.Log("CLIENT TAUNT");    
+            ClientManager.GetClient(clientId).Instance.GetComponent<Animator>().SetBool("Taunt", true);
```
Comment (MattiaPezzanoAiv) [inline Assets/Lobby-NetworkClient/ClientUtilities.cs]: use animator variable "Taunt" as integer.

File: Assets/Lobby-NetworkClient/ClientUtilities.cs [hunk 2]
```diff
@@ -152,14 +152,19 @@ public static Packet SendDodge(Vector3 pos, Quaternion rotation)
         Debug.Log("dodge sended instantly");
         return packet;
     }
-
-    public static Packet SendTaunt()
+    public static Packet SendTaunt(Vector3 pos, Quaternion rotation)
```
Comment (MattiaPezzanoAiv) [inline Assets/Lobby-NetworkClient/ClientUtilities.cs]: need to implement taunt command. (1 to 4)

File: Assets/Resources/aaaaaatry/PLAYER.prefab [hunk 3]
```diff
@@ -2289,10 +2289,10 @@ TextMesh:
   m_OffsetZ: 0
   m_CharacterSize: 1
   m_LineSpacing: 1
-  m_Anchor: 0
+  m_Anchor: 4
   m_Alignment: 0
   m_TabSize: 4
-  m_FontSize: 0
+  m_FontSize: 150
```
Comment (MattiaPezzanoAiv) [inline Assets/Resources/aaaaaatry/PLAYER.prefab]: remove text mesh from player

**Comment by aleaiudi** [comment]
now it should works

**Pull request merged**